content/applications/hr/time_off/my_time.rst

```diff
@@ -0,0 +1,106 @@
+=======
+My time
+=======
+
+The :guilabel:`My Time` section of the *Time Off* application houses all the various time off
```

Comment:
```suggestion
The *My Time* menu of the *Time Off* application houses all the various time off
```
Opting for italics here because technically the reader isn't on a page that shows this title... this sentence is basically a definition... and also the 'My Time' "section" is more of a "menu w/ options" -- so that's why I think italics > GUI in this situation